.github/workflows/nightly-testing.yaml

```diff
@@ -7,17 +7,19 @@ on:
     paths:
       - tasks/iac.yaml
       - .github/workflows/test-eks.yaml
-      - .github/bundles/*
-      - .github/test-infra/buckets-iac/*
-      - .github/test-infra/rds-iac/*
+      - .github/workflows/test-rke2.yaml
+      - .github/bundles/eks/*
+      - .github/bundles/rke2/*
+      - .github/test-infra/aws/eks/*
+      - .github/test-infra/aws/rke2/*
```

Comment:
More of a question/proposal - would it be valuable to move the `on.pull_request` triggers down a layer, into `test-eks` and `test-rke2`, rather than at this top level? If we did that PRs would be able to run just RKE2 or just EKS depending on what specific files were changed, reducing a bit of the load on some PRs if they only modified one or the other.

Not sure if there would be pieces where we might still want to run both based on though?

Reply:
We can save this for a future todo - I'll get an issue opened for it.

Reply:
Agree with the ask, can tackle this as a quick follow-up PR.